pmps.py

```diff
@@ -48,6 +47,8 @@ def __init__(self, parent=None, args=None, macros=None):
                 continue
             macros[m] = config.get(m)
 
+        # add CFG macro to display the Line when starting without macros
+        macros['CFG'] = config_name
```

Comment:
Hopefully this won't interfere with anything else?

Reply:
This probably won't interfere with anything